frontend/src/components/ProjectServiceAccountRow.vue

```diff
@@ -29,6 +29,12 @@ limitations under the License.
           >
             <template v-slot:popperRef>
               <span>{{displayName}}</span>
```

Comment:
disable the g-popper for invited service accounts, otherwise it looks like this

<img width="558" alt="Screenshot 2020-10-20 at 17 11 49" src="https://user-images.githubusercontent.com/5526658/96606560-b3efee80-12f7-11eb-84dd-e98eae55f19d.png">